atest/robot/cli/dryrun/executed_builtin_keywords.robot

```diff
@@ -13,3 +13,18 @@ Set Library Search Order
     Should Be Equal    ${tc.kws[1].name}    Second.Parameters
     Should Be Equal    ${tc.kws[2].name}    First.Parameters
     Should Be Equal    ${tc.kws[4].name}    Dynamic.Parameters
+
+Set Tags
+    ${tc} =    Check Test Case    ${TESTNAME}
+    List Should Contain Value    ${tc.tags}    Tag0
+    List Should Contain Value    ${tc.tags}    Tag1
+    List Should Contain Value   ${tc.tags}    Tag2
+    List Should Contain Value   ${tc.tags}   Tag3
+
+Remove Tags
+    ${tc} =    Check Test Case    ${TESTNAME}
+    List Should Contain Value   ${tc.tags}    Tag1
+    List Should Not Contain Value   ${tc.tags}    Tag2
+    List Should Contain Value   ${tc.tags}   Tag3
+
+
```

Comment:
`Check Test Tags` could be used here as well.

Extra empty lines at the end of the file should be removed. There should, however, be a new line at the end of the file.

Reply:
Updated!